opendebates/settings.py

```diff
@@ -8,6 +8,39 @@
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 FIXTURE_DIRS = [os.path.join(BASE_DIR, 'fixtures')]
 
+SITE_ID = 1
+SITE_DOMAIN = os.environ.get("SITE_DOMAIN", "127.0.0.1:8000")
+SITE_DOMAIN_WITH_PROTOCOL = os.environ.get("SITE_PROTOCOL", "http://") + SITE_DOMAIN
+
+SITE_THEMES = {
+    'florida': {
+        "HASHTAG": "FLOpenDebate",
+        "HEADER_TITLE": _("WELCOME TO THE\nFLORIDA OPEN DEBATE"),
```

Comment:
@ejucovy that's an interesting idea! But, I think we want to lean away from that approach. Using the i18n system for another purpose isn't something we have the time to test properly enough to feel comfortable about the support requirements. Keeping the distinctions between the site in this form also lets us keep the differences more clear in one place, and will let us adjust things other than test.